api/controllers/UserController.js

```diff
@@ -331,7 +331,7 @@ module.exports = {
           res.json('ok', 200);
         }, function(error) {
           console.log(error);
-          res.json({error: error}, 500);
+          res.json(error, 500);
         });
       });
     });
```

Comment:
No, don't fix it this way. Fix the error message on the client to show body.error rather than body. We should always return JSON properly.